test/fixtures/rules/std_rule_cases/L007.yml

```diff
@@ -55,3 +96,21 @@ passes_on_after_override:
     from foo
 
   configs: *operator_before
+
+fails_on_before_override_with_comment_order:
+  fail_str: |
+    select
+        a
+        -- comment1!
+        -- comment2!
+        + b
+    from foo
+  fix_str: |
+    select
+        a +
+        -- comment1!
+        -- comment2!
+        b
+    from foo
+
```

Comment:
Could we remove all newlines from test cases (including the ones you have not touched)?

Since we use a single newline between test cases I always find the other single newlines distracting and have been on a mission to slowly clean them up (I really should just do a bulk clean up!).

Reply:
Which new lines, the ones between yaml keys or the ones within tested SQL?

The later are kind of required for the test and rules. I imagine you mean the former?

Reply:
Lines 56, 97 and 115. They look like they are part of a separate test rather than continuation of the previous test if you get what I mean?